### DISCUSSION TITLE
Question about workspaces that seems to be the opposite of bumpVersionsWithWorkspaceProtocolOnly

### DISCUSSION THREAD

**Discussion by patrickconroy** [changesets/changesets#1101] [General]
Hello. We have a monorepo with a bunch of dependent packages using the `workspace:` protocol. When I run a `version` on something that's changed I don't want the packages who depend on them to bump/update in any way. It seems weird that they would while using `workspace` because your code is always local.

 E.g.

```json
{
  "name": "pkg1",
  "version": "1.0.0",
  "pkg2": "workspace:*"
}
```

Update pkg2, add a changeset that bumps pkg2, then run version, pkg1 will be bumped, even though the dep graph would be the same if it's bumped or not (since pkg2 is local).

```json
{
  "name": "pkg1",
  "version": "1.0.1",
  "pkg2": "workspace:*"
}
```

This is an issue because we have a monorepo with ~50 packages with inter-mingled dependency graphs. When a utility package gets bumped it might cause 30 other packages to get bumped. And with those bumps we test/build. Some people probably need that, but we don't (it's a microfrontend system). I'd love a config option that was like `dontBumpWorkspaceDependencies: boolean`.

Has anyone else found a way to do what I'm talking about?

**Comment by multivoltage** [comment]
did you solve it?
